ramls/circulation.raml

```diff
@@ -278,8 +278,8 @@ resourceTypes:
             description: "Patron type id"
             pattern: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
             required: true
-          shelving_location_id:
-            description: "Shelving location id"
+          location_id:
+            description: "Location id"
```

Comment:
This change is good.
This change requires a ModuleDescriptor-template.json version bump:

    "id": "circulation",
    "version": "7.11",

needs to bump to 8.0.
Consider moving the /circulation/rules /circulation/rules/* endpoints into a separate section, for example

    "id": "circulation-rules",
    "version": "1.0",

Reply:
Thank you! I changed it.